scripts/commands/addcurrency.lua

```diff
@@ -9,33 +9,125 @@ cmdprops =
     parameters = "sis"
 };
 
-function onTrigger(player,currency,amount,target)
-    local targ = nil
+function error(player, msg)
+    player:PrintToPlayer(msg);
+    player:PrintToPlayer("@addcurrency <currency type> <amount> {player}");
+end;
 
-    if (currency == nil or amount == nil) then
-        player:PrintToPlayer( "You must enter a valid currency and amount." );
-        player:PrintToPlayer( "@addcurrency <currency type> <amount> <target player>" );
-        return
-    end
+function onTrigger(player,currency,amount,target)
+    local currencies = {
+        sandoria_cp=true,
```

Comment:
i don't really like the idea of having to maintain the currency list in two places

the binding should be able to check if it errored and pass nil back